repositories.bzl

```diff
@@ -70,12 +70,57 @@ def com_google_api_codegen_repositories():
         urls = ["https://zlib.net/zlib-1.2.11.tar.gz"],
     )
 
+    _maybe(
+        http_archive,
+        name = "com_google_googleapis",
+        strip_prefix = "googleapis-84c8ad4e52f8eec8f08a60636cfa597b86969b5c",
+        urls = [
+            "https://github.com/googleapis/googleapis/archive/84c8ad4e52f8eec8f08a60636cfa597b86969b5c.zip",
+        ],
+    )
+
     _maybe(
         native.bind,
         name = "zlib",
         actual = "@net_zlib//:zlib",
     )
 
+    _maybe(
+        native.maven_jar,
+        name = "guava_maven",
```

Comment:
> but will it fail if compiled with newer guava (i.e. if you put on L97 "@com_google_guava_guava__com_google_api_codegen//jar" instead, will it break)?

Just tried this and the build still works! Gonna roll with this for now.

> Is there a better way to parse protobuf jsons?

No, not that I can find. It is the only lib I can find that will read JSON-pb format into the corresponding Message builder.

Reply:
I'm a bit confused here. The comment says build still works without this, but it is still here. I would not be surprised if it broke, but otherwise, please clean this up (the idea is to keep the dependencies as clean as possible, because they become a nightmare, when gapic-generator is used in wider scope and/or have to be kept up to date with its dependencies).

Reply:
This comment is on an outdated diff. There is no `guave_maven` dependency in `repositories.bzl`, just the `@com_google_guava_guava__com_google_api_codegen//jar"` you suggested I use [here](https://github.com/googleapis/gapic-generator/pull/2859/files#diff-515bc54a0cbb4b12fb4a7c465758b011R88-R92).

Reply:
@noahdietz Oh, sorry, you are right, I looked at a wrong dif. LGTM